source/CardGames.py

```diff
@@ -422,7 +424,11 @@ def check_illegal_card(self):
         else:
           #Check if hideout 
           if ((card - previous_hideout) <= 3) and ((card - previous_hideout) != 0):
-            self.cards_in_play.append(card)
+            real_cards = []
```

Comment:
Great work on this function George! For clarification, what is this real_cards list used for?

Reply:
Must have been a left over when troubleshooting. Thanks for catching that! It has been removed.